webaccess/src/webaccess.cpp

```diff
@@ -2081,6 +2089,66 @@ QString WebAccess::getChildrenHTML(VCWidget *frame, int pagesNum, int currentPag
     return unifiedHTML;
 }
 
+void WebAccess::slotGrandMasterValueChanged(uchar value)
+{
+    GrandMaster::ValueMode value_mode = m_vc->properties().grandMasterValueMode();
+    QString g_value_show;
```

Comment:
Code style: we don't use word_one_two_three in this code. We use valueOneTwoThree.
Also, variables should have a self-explaining meaning. In this case: gmDisplayValue